library/src/main/java/com/dslplatform/json/processor/Analysis.java

```diff
@@ -1019,6 +1028,29 @@ private void analyzeAttribute(
 			CompiledJson.TypeSignature typeSignature = typeSignatureValue(annotation);
 			JsonAttribute.IncludePolicy includeToMinimal = includeToMinimalValue(annotation);
 
+			boolean notNull = hasNonNullable(element, field, annotation);
+			boolean mandatory = hasMandatoryAnnotation(element, annotation) || field != null && hasMandatoryAnnotation(field, null);
+			if (converter != null && converter.binder != null) {
+				// if value is null, the binding value will be lost
+				if (!notNull) {
+					hasError = true;
+					messager.printMessage(
+							Diagnostic.Kind.ERROR,
+							"Attribute with binder can not be nullable on " + name + " in " + info.name,

```

Comment:
I think it should be fine that its nullable, just use different method for processing in that case